crates/sui-types/src/move_package.rs

```diff
@@ -45,6 +45,7 @@ pub type FnInfoMap = BTreeMap<FnInfoKey, FnInfo>;
 #[derive(Eq, PartialEq, Debug, Clone, Deserialize, Serialize, Hash)]
 pub struct MovePackage {
     id: ObjectID,
+    version: SequenceNumber,
```

Comment:
Just to double check from the summary of the PR, this is a pun for everything but the framework/stdlib correct? 
In other words, for non framework packages, we will never have two packages with the same id but different versions? 

Might be worth commenting here just since its different than how it works for MoveObjects

Reply:
Yep, that's correct.  Only the framework/stdlib can exist at multiple versions for the same ObjectID but all packages will store their package version (rather than the Lamport version of their parent transaction) in this field.